.gitignore

```diff
@@ -34,3 +34,6 @@ node_modules
 # Optional REPL history
 .node_repl_history
 
+.env
+
+package-lock.json
```

Comment:
Should this file be ignored?

Reply:
Yes I think it can be ignored because we have a package.json

Reply:
Research more about this issue.